### DISCUSSION TITLE
Roomba create 3 no longer pairing when i click butons 1 & 2

### DISCUSSION THREAD

**Discussion by adnanwahab** [iRobotEducation/create3_docs#460] [Questions]
### How are you connecting to your Create 3?

Wi-Fi and USB Ethernet (ROS 2)

### Which version of ROS 2 is installed on your computer?

Galactic

### Which firmware version is installed on your robot?

not sure

### Which RMW is your robot running?

FastRTPS

### Does your robot have an assigned namespace? If so, please share.

none

### Is the robot connected to a network? If so, what is the network type?

no it is no longer connecting

### Are there multiple Create 3 robots connected to your network?

No

### Is multicast enabled?

No

### What is the Adapter Board's USB/BLE Toggle currently switched to?

Bluetooth (default)

### Describe your question.

Hi my roomba create 3 will no longer pair with my macbook. it was working a few days ago and i could use the irobot playground https://python.irobot.com/. 

But now when i click the buttons 1 & 2, and hold them for 20 seconds, it no longer connects to bluetooth and no longer shows the blue light. Please help. How can i restore my roomba create 3 to factory settings?

**Comment by kwujciak** [comment]
You may not need to reset to factory settings. Holding buttons 1 and 2 activates the robot's access point, it doesn't "start" the Bluetooth. It's like the robot's own network that just allows you change settings on your robot, check logs, update firmware, etc. If you were able to connect via Bluetooth a few days ago, I'm assuming all your settings are fine. In other words, you do not need to activate the robot's access point in order to connect via Bluetooth. First, I would double check that your mac bluetooth is on. Also, make sure you're accessing the python playground on Google Chrome, not Safari. 

You do not need to enter the robot's access point to connect to bluetooth (i.e. you don't need to hold down buttons 1 and 2). Just make sure your robot is on (place on the doc), bluetooth is enabled on your Mac, and you're on Chrome.

**Comment by shamlian** [comment]
I'm likewise confused about this and not sure exactly what state your robot and computer are in. As @kwujciak said above, pushing (1) and (2) don't affect Bluetooth connectivity; instead, that combination is used to connect your robot to a Wi-Fi network. If you're still having trouble and the robot is not responding, I'd recommend ensuring the switch on the [adapter board](https://iroboteducation.github.io/create3_docs/hw/adapter/) is set to Bluetooth and then forcing the robot to hard-reboot by removing it from the dock and holding down the center button for thirty seconds in order to get the robot into a known good state.

Have you updated your robot or connected it to Wi-Fi in the past? What version of firmware is it running? And what version of OS X do you have?
